crates/re_types/.gitattributes

```diff
@@ -0,0 +1,35 @@
+.gitattributes linguist-generated=true
+src/archetypes/fuzzy.rs linguist-generated=true
+src/archetypes/mod.rs linguist-generated=true
+src/archetypes/points2d.rs linguist-generated=true
+src/archetypes/points3d.rs linguist-generated=true
+src/archetypes/transform3d.rs linguist-generated=true
+src/components/class_id.rs linguist-generated=true
+src/components/color.rs linguist-generated=true
+src/components/draw_order.rs linguist-generated=true
+src/components/fuzzy.rs linguist-generated=true
+src/components/instance_key.rs linguist-generated=true
+src/components/keypoint_id.rs linguist-generated=true
+src/components/label.rs linguist-generated=true
+src/components/mod.rs linguist-generated=true
+src/components/point2d.rs linguist-generated=true
+src/components/point3d.rs linguist-generated=true
+src/components/radius.rs linguist-generated=true
+src/components/transform3d.rs linguist-generated=true
+src/datatypes/angle.rs linguist-generated=true
+src/datatypes/fuzzy.rs linguist-generated=true
+src/datatypes/mat3x3.rs linguist-generated=true
+src/datatypes/mat4x4.rs linguist-generated=true
+src/datatypes/mod.rs linguist-generated=true
+src/datatypes/point2d.rs linguist-generated=true
+src/datatypes/point3d.rs linguist-generated=true
+src/datatypes/quaternion.rs linguist-generated=true
+src/datatypes/rotation3d.rs linguist-generated=true
+src/datatypes/rotation_axis_angle.rs linguist-generated=true
+src/datatypes/scale3d.rs linguist-generated=true
+src/datatypes/transform3d.rs linguist-generated=true
+src/datatypes/translation_and_mat3x3.rs linguist-generated=true
+src/datatypes/translation_rotation_scale3d.rs linguist-generated=true
+src/datatypes/vec2d.rs linguist-generated=true
+src/datatypes/vec3d.rs linguist-generated=true
+src/datatypes/vec4d.rs
```

Comment:
why is this guy different? :disappointed:

Reply:
huh. good catch. that's because `join(..)` is not what I wnated here!

Reply:
fixed